drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -62,6 +63,12 @@ def self.sort_by_option(option)
     end
   end
 
+  def self.project_search(input:, user: nil)
+    scope = Hmis::Hud::Project.where(id: viewable_by(user).select(:id))
```

Comment:
Can the scope just be `Hmis::Hud::Project.viewable_by(user)`? Also I think `user` should be a required argument here. I don't think we ever want to do a search that's not scoped by permissions.

Reply:
I agree.

Reply:
Fixed